### PR TITLE
Change wording of available interfaces in debug script

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
Fixes https://github.com/pi-hole/pi-hole/issues/2945

**How does this PR accomplish the above?:**
Only change wording

### CODE REVIEW

File: advanced/Scripts/piholeDebug.sh [hunk 1]
```diff
@@ -618,7 +618,7 @@ detect_ip_addresses() {
         # Local iterator
         local i
         # Display the protocol and interface
-        log_write "${TICK} IPv${protocol} address(es) bound to the ${PIHOLE_INTERFACE} interface:"
+        log_write "${TICK} IPv${protocol} address(es) available to the ${PIHOLE_INTERFACE} interface:"
```
Comment (DL6ER) [inline advanced/Scripts/piholeDebug.sh:621]:
```suggestion
        log_write "${TICK} IPv${protocol} address(es) attached to the ${PIHOLE_INTERFACE} interface:"
```

**Review by DL6ER — CHANGES_REQUESTED** [review]
Well, it's not a mistake as such. The OP of the references issue said:

> But the log says these addresses are bound to `pihole-FTL`.

No, it doesn't. This is just their own interpretation as they admit a bit further down:

> I read this as the addresses that are also bound to FTL, cause that's the only reason for the data collection.

---

I'm not convinced an address can be "available" to an interface. It really is "bound" to the interface. Note: this has nothing to do with any process possibly bound to the addresses as well.

`man ip address` says:
``` plain
DESCRIPTION
       The address is a protocol (IPv4 or IPv6) address **attached** to a network
       device. Each device must have at least one address to use the corre‐
       sponding protocol. It is possible to have several different addresses
       attached to one device. These addresses are not discriminated, so that
       the term alias is not quite appropriate for them and we do not use it
       in this document.

       The ip address command displays addresses and their properties, adds
       new addresses and deletes old ones.
```

Hence, if we make this change at all, I'd vote for "attached" instead of "available". However, "attached" is that close to "bound" that the change may not be needed but I'll not vote against it.

Feedback by others wanted, too.

File: advanced/Scripts/piholeDebug.sh (hunk 1 above)
Comment (jfb-pihole) [reply advanced/Scripts/piholeDebug.sh:621]: Bound is a common term.  Apps bind to ports and interfaces.

**Comment by dschaper** [comment]
I have no opinion on it. How ever you'd like to see it. `bound` was fine for me.

**Comment by yubiuser** [comment]
I'm fine with leaving it as `bond` too.  Going to close this PR and the issue.

**Pull request closed without merging**